content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
+The default options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
 :guilabel:`Basic Wage for Time Off`, :guilabel:`Days of Paid Time Off`, :guilabel:`Days of
 Unforeseen Absence`, :guilabel:`Days of Unpaid Time Off`, :guilabel:`Gross Wage`, :guilabel:`Net
 Wage`, :guilabel:`Number of Days`, :guilabel:`Number of Hours`, :guilabel:`Work Days`,
 :guilabel:`Work Hours`, and :guilabel:`Count`.
 
+:guilabel:`Net Wage` is the default metric for the :guilabel:`Payroll` report.
+
 .. image:: reporting/measures.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: The various measures available to display for the Payroll Analysis report.
 
 Line chart
 ~~~~~~~~~~
 
-A line chart is the default view for the *Payroll Analysis* report. If a different view was
-selected, click on the :guilabel:`Line Chart` icon (middle icon) in the menu bar to change the view
-back to a line chart. An option to display the line chart in ascending or descending order appears
-at the end of the options.
+A line chart is the default view for the *Payroll* report. If a different view is selected, click
+the :guilabel:`Line Chart` icon in the menu bar to change the view back to a line chart.
+
+Several options are available for the line chart. Click the corresponding button to activate the
+selection, and change the way the data is presented.
+
+The various options are: :guilabel:`Stacked`, :guilabel:`Cumulative`, or in :guilabel:`Descending`
+or :guilabel:`Ascending` order. These icons appear at the end of the chart options.
 
 .. image:: reporting/line-chart.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Menu buttons with the line chart called out, along with the other option buttons.
 
 Bar chart
 ~~~~~~~~~
 
-To display the data in a bar chart, click on the :guilabel:`Bar Chart` icon (first icon) in the menu
-bar.
+To display the data in a bar chart, click on the :guilabel:`Bar Chart` icon in the menu bar.
+
+Click the :guilabel:`Stacked` icon to view the bar chart in a stacked format (where multiple values
+appear in each column). An option to display the columns in :guilabel:`Descending` or
+:guilabel:`Ascending` order appears at the end of the options.
 
 .. image:: reporting/bar-chart.png
    :align: center
-   :alt: Report dashboard view.
-
-Click the :guilabel:`Stacked` icon to view the bar chart in a stacked format (where multiple values
-appear in each column). An option to display the columns in ascending or descending order appears at
-the end of the options.
+   :alt: Menu buttons with the bar chart called out, along with the other option buttons.
 
 .. tip::
-   Clicking an option enables it. To disable the option, click it again. When the option is enabled
-   it appears grey. When it is inactive, it appears white.
+   Clicking an option enables it. To turn off the option, click it again. When the option is enabled
+   it appears gray. When it is inactive, it appears white.
```

Comment:
I'd argue that the converse is true about the colors...it appears to lighten up (white) when active, and become purely gray when inactive

Reply:
Good catch! I changed it even further since it has a turquoise outline as well, so it is very clear now.